addon/content/js/handlebars-v3.0.0.js

```diff
@@ -209,6 +209,27 @@ var __module2__ = (function(__dependency1__, __dependency2__) {
     registerDefaultHelpers(this);
   }
 
+  var logger = {
```

Comment:
I don't want to modify this files - it is vendor imported at a specific version, and if anything we should be fixing upstream and updating it.

In this case, I'm working on a branch that will remove this completely that I hope to land in a month or two. Seeing as there's nothing critical here, I think we should just drop the changes to this file.

Reply:
Thanks, done